### PR TITLE
[595] feat: add website screenshot to directory

### PR DESCRIPTION
**Pull request by mvstepanek**
**Changes proposed in this Pull Request**
Add URLsLab screenshot to Directory
Add link to company website in perex

**Testing instructions**
E.g. [this URL](http://liveagent.local/directory/airbus-help-desk-contacts/).

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

Close QualityUnit/web-issues#595

### CODE REVIEW

File: assets/styles/pages/Directory.scss [hunk 1]
```diff
@@ -5,6 +5,14 @@
 
 .Directory {
 
+	&__screenshot {
+		margin-bottom: 45px;
+
+		a::after {
+			display: none !important;
```
Comment (jremen) [inline assets/styles/pages/Directory.scss:12]: If important not required, do not use it

File: assets/styles/pages/Directory.scss [hunk 2]
```diff
@@ -5,6 +5,14 @@
 
 .Directory {
 
+	&__screenshot {
+		margin-bottom: 45px;
```
Comment (jremen) [inline assets/styles/pages/Directory.scss:9]: Use relative units

**Review by jremen — CHANGES_REQUESTED** [review]
Replace pixels with EMs

File: assets/styles/pages/Directory.scss (hunk 1 above)
Comment (mvstepanek) [reply assets/styles/pages/Directory.scss:12]: Unfortunately, it is necessary.

File: assets/styles/pages/Directory.scss (hunk 2 above)
Comment (mvstepanek) [reply assets/styles/pages/Directory.scss]: Fixed.

**Review by jremen — DISMISSED** [review]

**Review by jremen — APPROVED** [review]

**Pull request merged**